Tasks/HelmDeployV1/src/helm.ts

```diff
@@ -61,20 +61,18 @@ async function getKubeConfigFile(): Promise<string> {
 }
 
 async function runHelmSaveCommand(helmCli: helmcli, kubectlCli: kubernetescli, failOnStderr: boolean): Promise<void> {
-    if (!helmCli.isHelmV3()) {
-        //helm chart save and push commands are only supported in Helms v3  
-        throw new Error(tl.loc("SaveSupportedInHelmsV3Only"));
+    if (!helmCli.isHelmV37Plus()) {
+        //helm chart commands are removed in Helms v3.7+
+        throw new Error(tl.loc("PackageSupportedInHelmsV37Only"));
```

Comment:
Here checks helm version, it will throw an exception if helm version less or equal to 3.7. HelmDeployV1 would require helm 3.7.0+ to run commands like helm push. Helm package (previously helm chart save command) supported since 3.0 though.